app/features/connect_with_them/Credentials.py

```diff
@@ -0,0 +1,86 @@
+from dotenv import load_dotenv, find_dotenv
+import os
+
+load_dotenv(find_dotenv())
+
+credentials =   {
+        'CLIENT_ID' : os.getenv('CLIENT_ID'),
+        'CLIENT_SECRET' : os.getenv('CLIENT_SECRET'),
+        'NAME' : os.getenv('NAME'),
+        'PASSWORD' : os.getenv('PASSWORD'),
+        'USER_AGENT' : os.getenv('USER_AGENT'),
+        'GROQ_API_KEY' : os.getenv('GROQ_API_KEY')
+}
+
+categories_mapping = {
```

Comment:
As we're working with commercial LLMs, this mapping can limit our extention with the range of domain models that we want to reach. Nevertheless, can be certainly beneficial for local LLMs, but a good practice is to generalize this in order to satisfy more use cases.